### DISCUSSION TITLE
Default page for docs home page

### DISCUSSION THREAD

**Discussion by saeedesmaili** [fastapi-users/fastapi-users#1198] [General]
Sorry if this is not an issue to be mentioned here.
I wonder if it's possible to redirect the home page of the docs (https://fastapi-users.github.io/) to the latest version's doc (https://fastapi-users.github.io/fastapi-users/10.4/ for example) automatically? I frequently open https://fastapi-users.github.io/ and 404 page reminds me that I should try to find the correct url in my browser history or on Google.

**Comment by frankie567** [comment]
Nice suggestion! I didn't know it was possible to build a site for the root domain with GitHub Pages. I've just setup a simple redirection; that should do the trick!

**Reply by saeedesmaili** [reply]
Yes! It works as expected now.
